requests-for-comments/2022-12-01-workflow-new-feature.md

```diff
@@ -0,0 +1,57 @@
+# The first "Workflow": MVP for "Creating a new feature"
+
+Owner: @EDsCODE (team experimentation)
+Decision to be made: Should we build this in Q1 as part of team experimentation?
+Decision date: 7th December 2022
+
+## Context
+
+This RFC proposes that team experimentation builds the MVP of a workflow for "Creating a new feature" as part of their Q1 OKRs.
+
+It should sit with team experimentation as it would be a key driver of feature flag usage.
+
+This could have multiple benefits across the product:
```

Comment:
The promise for an opinionated workflow makes a lot of sense. If we can properly automate the pre and post implementation for engineers this would be a significant value add.

**I will asterisk that adding opinionated features like this could get confusing from a marketing standpoint. Most of the product is fairly unopinionated "here's analytics tooling you can use to understand your users" while this feature moves in the direction "here's a product _process_"

Reply:
Could we give them the tools to customize the create a new feature workflow? E.g. they can edit the schema which decides which graphs, session recordings, feature flag setups are made. We could use the new JSON based stuff that Marius is making with some template variables

This way we can give them some starter workflows but they can also edit it or create their own to suit their processes